### DISCUSSION TITLE
List Filter with Mine

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14329] [How to] [answered]
We want to create filters for user where assigned to should be filtered for logged in user i.e. Mine

How do I do this ?

![image](https://user-images.githubusercontent.com/12723595/110095564-10ddd400-7dc3-11eb-8d1e-53f7be6d9771.png)

Here in this list do not have option to keep it mine. due to this we need to create filter for each user to show only his opportunities. Providing such option will make it global

Regards
Jeevan Pendhari

**Comment by vovpff** [comment]
I'm also interested!

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10198654/110117221-dbcc8400-7db8-11eb-98ff-657f5d46ba89.png)

**Reply by JEEVAN28** [reply]
Great !!!
